.gitignore

```diff
@@ -53,6 +53,6 @@ dkms.conf
 
 # Clion files
 .idea
-CMakeLists.txt
+#CMakeLists.txt
```

Comment:
This one actually will be comment only for my personal branch since I am using the CMakeLists.txt now, but for master it should the uncommented so it will be ignored. I will uncomment it after the pull request is accepted